Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/Constants.java

```diff
@@ -57,17 +57,17 @@ public class Constants {
     public static final int CAMERA_CENTER_X = CAMERA_X / 2;
     public static final int CAMERA_CENTER_Y = CAMERA_Y / 2;
 
-    public static final int CONE_WIDTH = 350;
+    public static final int CONE_WIDTH = 400;
     public static final double CONE_STACK_WIDTH_KP = -0.00005;
     public static final double CONE_STACK_CENTERING_KP = -0.0005;
-    public static final double CONE_STACK_CENTERING_PROPORTIONAL_KP = -0.0003;
-    public static final double CONE_STACK_CENTERING_MAX_SPEED = 0.25;
+    public static final double CONE_STACK_CENTERING_PROPORTIONAL_KP = -0.00042;
+    public static final double CONE_STACK_CENTERING_MAX_SPEED = 0.27;
 
     public static final int JUNCTION_TOP_TOLERANCE = 100;
     public static final double JUNCTION_TOP_CENTERING_KP = 0.0004;
 
-    public static final Scalar LOWER_RED = new Scalar(20, 0, 0);
-    public static final Scalar UPPER_RED = new Scalar(160, 255, 252);
+    public static final Scalar LOWER_RED = new Scalar(17, 0, 0);
+    public static final Scalar UPPER_RED = new Scalar(165, 150, 30);
```

Comment:
Does this have the new red constants? Make sure to close the other PR once this is updated.